pkg/network/dns/resolveconf.go

```diff
@@ -122,3 +126,29 @@ func DomainNameWithSubdomain(searchDomains []string, subdomain string) string {
 
 	return ""
 }
+
+// returns nameservers [][]byte, searchdomains []string, error
```

Comment:
Now that this is moved to a maintained location, please fix the function doc text.

Reply:
Done.